PendingReleaseNotes

```diff
@@ -22,6 +22,20 @@
   support an NFS export of both ``rgw`` and ``cephfs`` from a single
   NFS cluster instance.
 
+* The ``nfs cluster update`` command has been removed.  You can modify
+  the placement of an existing NFS service (and/or its associated
+  ingress service) using ``orch ls --export`` and ``orch apply -i
+  ...``.
```

Comment:
@sebastian-philipp @varshar16 We need to make a call here on whether to (1) remove this, (2) keep it but require that all relevant arguments be provided (i.e., wrap ``orch apply nfs`` and do not support incremental changes/patches), or (3) implement a more complex version that allows incremental changes.

I lean toward (1) since it's the simplest and this is convenience wrapper only, and because we can always come back later with something more friendly if we see a need.

Reply:
How about making it a wrap around `orch apply -i nfs.yaml`  instead of `orch apply nfs` ? We would need additional checks to ensure the yaml is related to `nfs` service only.

Reply:
The calls to `orch apply -i nfs.yaml` and `orch apply nfs` are identical. I'd be ok with (1)